olca-app/src/org/openlca/app/editors/ModelPage.java

```diff
@@ -35,7 +35,7 @@ public ModelPage(ModelEditor<T> editor, String id, String title) {
 	}
 
 	public String getFormTitle() {
-		return getTitle() + ": " + Labels.name(getModel());
+		return String.join(" - ", getTitle(), Labels.name(getModel()));
```

Comment:
`String.join(" - ", A, B)` is not better readable than `A + " - " + B`; it is also _not_ more efficient as it does an array allocation each time (!)

Reply:
My idea was that if we want to change the way we join such strings in the future, it will be easier to search and find everything at once.
For example, we want to provide support for RTL languages. We just need to replace `String.join(" - "` with `OurOwnStringJoin(`. If we want to switch back to `: ` instead of ` - `, it is also quicker.

Should I revert?

Reply:
I would revert it, we could also just search for the `"M." * " + " * " - " * " + " * "M."` pattern and find these things, if we really want to change it (we should never implement something that reduces readability and performance because it could be potentially improved later, because that later often never happens or looks quite different then)